src/Uno.Toolkit.UI/Behaviors/InputExtensions.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Windows.System;
+using Windows.UI.ViewManagement;
+
+#if IS_WINUI
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+#else
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+#endif
+
+namespace Uno.Toolkit.UI
+{
+	public static class InputExtensions
+	{
+		#region DependencyProperty: AutoDismiss
+
+		public static DependencyProperty AutoDismissProperty { get; } = DependencyProperty.RegisterAttached(
+			"AutoDismiss",
+			typeof(bool),
+			typeof(InputExtensions),
+			new PropertyMetadata(default(bool), OnAutoDismissChanged));
+
+		public static bool GetAutoDismiss(DependencyObject obj) => (bool)obj.GetValue(AutoDismissProperty);
+		public static void SetAutoDismiss(DependencyObject obj, bool value) => obj.SetValue(AutoDismissProperty, value);
```

Comment:
Idem, please follow the standard by prefixing your `bool` properties with `Is`.

Reply:
We are keeping the existing property name _for now_.